RELEASING.md

```diff
@@ -0,0 +1,192 @@
+| cancun | devnet1 bugfix     | `1.15.0rc1.post3` |
+|        |                    |                   |
+| cancun | finalize devnet2   | `1.15.0rc2`       |
+|        |                    |                   |
+| cancun | finalize mainnet   | `1.15.0`          |
+|        |                    |                   |
+| cancun | mainnet bugfix     | `1.15.1`          |
+|        |                    |                   |
+| cancun | breaking change    | `2.15.0`          |
+|        |                    |                   |
+| prague | preview of devnet1 | `2.16.0rc1.dev1`  |
+|        |                    |                   |
+| prague | finalize devnet1   | `2.16.0rc1`       |
+|        |                    |                   |
+| prague | finalize mainnet   | `2.16.0`          |
+
+
+## Creating a Release
+
+### Overview
+
+1. Choose a version number.
+1. Update version in source code.
+1. Create a pull request.
+1. Wait for it to get merged.
+1. Create a tag.
+1. Create GitHub release.
+1. Publish to PyPI.
+
+### Choosing a Version Number
+
+To choose the next version number, find the format matching the current version
+number in the table below, then choose the new version according to the reason
+for the new release.
+
+| Current Version           | Action               | New Version            |
+| ------------------------- | -------------------- | ---------------------- |
+| **`1.3.5`**               |                      |                        |
+|                           | Mainnet Release      | `1.4.0`                |
+|                           | Devnet Release       | `1.4.0rc1`             |
+|                           | Bug Fix Release      | `1.3.6`                |
+|                           | Breaking Release     | `2.3.0`                |
+|                           |                      |                        |
+| **`1.3.0rc5`**            |                      |                        |
+|                           | Mainnet Release      | `1.3.0`                |
+|                           | Devnet Release       | `1.3.0rc6`             |
+|                           | Bug Fix Release      | `1.3.0rc5.post1`       |
+|                           | Breaking Release     | `2.3.0rc5`             |
+|                           |                      |                        |
+| **`1.3.0rc5.post7`**      |                      |                        |
+|                           | Mainnet Release      | `1.3.0`                |
+|                           | Devnet Release       | `1.3.0rc6`             |
+|                           | Bug Fix Release      | `1.3.0rc5.post8`       |
+|                           | Breaking Release     | `2.3.0rc5`             |
+|                           |                      |                        |
+| **`1.3.5.dev7`**          |                      |                        |
+|                           | Mainnet Release      | `1.3.5`                |
```

Comment:
As per the system, would all mainnet releases be *.*.0?

```suggestion
|                           | Mainnet Release      | `1.3.0`                |
```

Reply:
The path that leads here is:

* 1.3.0 <- first mainnet release 
* 1.3.1 <- mainnet bugfix
* 1.3.2
* 1.3.4
* 1.3.5.dev1 <- first preview of 1.3.5
* 1.3.5.dev2
* 1.3.5.dev3
* 1.3.5.dev4
* 1.3.5.dev5
* 1.3.5.dev6
* 1.3.5.dev7 <- last preview of 1.3.5
* 1.3.5 <- mainnet bugfix